README.md

```diff
@@ -10,15 +10,46 @@ This repository contains an embeddable reference interpreter, test framework, an
 The easiest way to get started with PartiQL is to clone this repository locally, build, then 
 [run the REPL](./docs/user/CLI.md).
 
+
 ## Status
 
-PartiQL should be considered to be in "preview" status.  It has been in use within a number of Amazon
-internal systems and an AWS product for over one year.  The behavior of the language itself is mostly stable 
-however the public API of the interpreter is slated to undergo significant improvements in the near term.
-(See the GitHub issues list for details.)
+PartiQL should be considered to be in "preview" status. It has been in use within a number of Amazon internal 
+systems and an AWS product for over one year. The behavior of the language itself is mostly stable however 
+the public API of the interpreter is slated to undergo significant improvements in the near term. (See the 
+GitHub issues list for details.)
+
+## Using In Your Project
+
+This project is published to [Maven Central](https://search.maven.org/artifact/org.partiql/partiql-lang-kotlin/0.1.0/jar).
+
+| Group ID | Artifact ID | Recommended Version |
+|----------|-------------|---------------------| 
+| `org.partiql` | `partiql-lang-kotlin` | `0.1.0`
+
+
+For Maven builds, add this to your `pom.xml`:
+
+```
+<dependency>
+  <groupId>org.partiql</groupId>
+  <artifactId>partiql-lang-kotlin</artifactId>
+  <version>{version}</version>
```

Comment:
Another option is to just make this (and the gradle dependency below) explicitly `0.1.0`. We're going to have to change this file for every release anyway due to the "Recommended Version" column above.

Reply:
I was trying to avoid having to update three different version numbers.